### PR TITLE
Update webpacker configuration

### PR DESCRIPTION
**Pull request by rhymes**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

As we're slowly approaching upgrading to Rails 6, I'm starting to set foundation stones in place.
This was (hopefully) an easy win.

Ran `rails webpacker:install` and merged the new configuration with the existing one.

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

### CODE REVIEW

File: config/webpack/test.js [hunk 1]
```diff
@@ -1,3 +1,7 @@
+/* global process, module, require */
+
+process.env.NODE_ENV = process.env.NODE_ENV || 'development';
```
Comment (nickytonline) [inline config/webpack/test.js:3]: Shouldn't this be `'test'`?

```suggestion
process.env.NODE_ENV = process.env.NODE_ENV || 'test';
```

Comment (rhymes) [reply config/webpack/test.js:3]: @nickytonline maybe? https://github.com/rails/webpacker/blob/master/lib/install/config/webpack/test.js

🤷

Comment (nickytonline) [reply config/webpack/test.js:3]: I asked a question, https://github.com/rails/webpacker/issues/2602, in their repo about this.

Comment (rhymes) [reply config/webpack/test.js:3]: I have the feeling the reason why it's working is that when you run webpacker it sets `NODE_ENV` correctly, so it never gets to the `OR` second clause

**Pull request merged**